src/core/common/debug.hpp

```diff
@@ -57,7 +57,7 @@
  */
 #ifndef FILE_NAME
 #define FILE_NAME __FILE__
-#endif
+#endif // FILE_NAME
```

Comment:
Per our style guide, we add the comment on `#endif` if the number of lines between `#if` and `#endif` is more than 20 lines. Would be better to not add it in cases like this (does not really help with code readability)
See this in style guide: 
https://github.com/openthread/openthread/blob/d65f8a3812611a89f851a18df2d4b8219a6fbca2/STYLE_GUIDE.md#conventions-and-best-practices

Reply:
Done.